From: Max Gurtovoy <maxg@mellanox.com>
Date: Thu, 27 Oct 2016 16:36:36 +0300
Subject: IB/mlx5: Replace numerical constant with predefined MACRO
Patch-mainline: v4.10-rc1
Git-commit: 2d2215888d758c194efb38332aaf5e2069ac578e
References: bsc#1015342 FATE#321688 bsc#1015343 FATE#321689

Replace the pre-defined macro signifying inline umr instead
of the numerical constant.

Signed-off-by: Max Gurtovoy <maxg@mellanox.com>
Signed-off-by: Leon Romanovsky <leon@kernel.org>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/infiniband/hw/mlx5/qp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/infiniband/hw/mlx5/qp.c
+++ b/drivers/infiniband/hw/mlx5/qp.c
@@ -3067,7 +3067,7 @@ static void set_linv_umr_seg(struct mlx5
 {
 	memset(umr, 0, sizeof(*umr));
 	umr->mkey_mask = cpu_to_be64(MLX5_MKEY_MASK_FREE);
-	umr->flags = 1 << 7;
+	umr->flags = MLX5_UMR_INLINE;
 }
 
 static __be64 get_umr_reg_mr_mask(void)
